src/react/README.md

```diff
@@ -67,3 +67,40 @@ const MyComponent = () => {
   return <div>Connection status is: {currentStatus}</div>;
 };
 ```
+
+## usePresence
+
+This hook allows you to access the `Presence` instance of a specific room from the nearest `ChatRoomProvider` in the
```

Comment:
I think we should be more specific earlier on in the documentation that this hook relates to enter/update operations with presence. I have to go down 3 paragraphs to deduce this - which I think is probably too far.

We should also direct people to the appropriate hook for listening to presence events.